test/data/import/new/full-export.json

```diff
@@ -17,6 +17,7 @@
                         "endpoint": "answer",
                         "documentation": "",
                         "enabled": true,
+                        "randomResponse": false,
```

Comment:
Please do not modify files in `test/data/migrations`, `test/data/import/new` and `test/data/import/old`